src/io/state-file/index.ts

```diff
@@ -18,19 +21,35 @@ import {
   retypeFile,
 } from '../io';
 import { FileEntry } from '../types';
-import { Manifest, ManifestSchema, DataSet } from './schema';
+import { Manifest, ManifestSchema } from './schema';
 
 const MANIFEST = 'manifest.json';
 const VERSION = '0.0.1';
 
 export async function save(fileName: string) {
   const datasetStore = useDatasetStore();
   const viewStore = useViewStore();
+  const labelStore = useLabelmapStore();
+  const toolStore = useToolStore();
 
   const zip = new JSZip();
   const manifest: Manifest = {
     version: VERSION,
     dataSets: [],
+    labelMaps: [],
+    tools: {
+      rulers: [],
+      crosshairs: {
+        position: [0, 0, 0],
+      },
+      paint: {
+        activeLabelmapID: null,
+        brushSize: 8,
+        brushValue: 1,
+        labelmapOpacity: 1,
+      },
+      current: Tools.WindowLevel,
```

Comment:
Does it make sense to save the current tool? I think the data is the most important thing. (This also means we don't need to save the crosshairs state, since it will be overwritten the next time it's activated anyways.) @aylward

Reply:
I suppose this doesn't matter for now. Saving this state is mostly inconsequential. We can adjust later as needed.

Reply:
Personally I think saving the current tool is nice, as it more complete restores the state of your previous session, however, I can be persuaded to remove it.